Storage/SQL/BrowserDB.swift

```diff
@@ -57,7 +57,7 @@ enum TableResult {
 public class BrowserDB {
     private var db: SwiftData
     // XXX: Increasing this should blow away old history, since we currently don't support any upgrades.
-    private let Version: Int = 7
+    private let Version: Int = 8
```

Comment:
I wonder if we should add a Github label to indicate that this PR will cause this effect?